### PR TITLE
add null safety

### PR DESCRIPTION
**Pull request by AntonyLeons**
as part of flutter 2.0

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name: country_codes
 description: Country codes is an helper package that provides country details given a particular localization, such as dial codes, ISO 3166 codes (alpha-2, alpha-3) and country names.
-version: 1.0.3
+version: 1.1.0
```
Comment (miguelpruivo) [inline pubspec.yaml:3]: This should be bumped to major (`2.0.0`).

**Review by miguelpruivo — CHANGES_REQUESTED** [review]
Thank you. Didn't remember to upgrade this yet. Please update the major version.

File: pubspec.yaml (hunk 1 above)
Comment (AntonyLeons) [reply pubspec.yaml]: done

**Comment by AntonyLeons** [comment]
looks good, please test and merge

**Review by miguelpruivo — APPROVED** [review]

**Comment by miguelpruivo** [comment]
Thank you!

**Comment by Ponjimon** [comment]
When will this change be published to pub.dev?

**Comment by miguelpruivo** [comment]
Today 😬

**Comment by miguelpruivo** [comment]
Done. Thank you!

**Pull request merged**